scripts/express/app.js

```diff
@@ -1,12 +1,11 @@
-/* eslint-disable import/no-extraneous-dependencies */
 /* eslint-disable no-console */
 const express = require('express');
 const path = require('path');
 
 const app = express();
 const port = process.env.PORT || 8081;
 // path to webpack built path
-const buildPath = path.join(__dirname, '..', '..', 'build');
+const buildPath = path.join(process.cwd(), '..', '..', 'build');
```

Comment:
Actually, i don't think you want to go down from the process.cwd() dir. Probably just 
`path.join(process.cwd(), 'build')`